docs/developer/checkout/address.mdx

```diff
@@ -229,6 +229,14 @@ As a result, we get an updated checkout object with a delivery method set:
 }
 ```
 
+### Switching from C&C to standard delivery method
+
+After choosing the click & collect option on a checkout, going back to a standard delivery method will require the shipping address to be set, this can't be done directly by checkoutDeliveryMethodUpdate, but can be achieved using the flow below.
+
+1. First use `checkoutDeliveryMethodUpdate` (just with `id` or `token` provided). This will remove the collection point from checkout and also remove the shipping address.
```

Comment:
IMO:
- we should add link to the API docs for this mutation: https://docs.saleor.io/docs/3.x/api-reference/checkout/mutations/checkout-delivery-method-update
- we should add graphl example for that like:
```
mutation {
  checkoutDeliveryMethodUpdate(id: "your_checkout_id") {
    checkout {
      id
    }
  }
}
```